frontend/src/views/IssueDetail.vue

```diff
@@ -263,12 +301,28 @@ export default defineComponent({
       return false;
     };
 
+    const maybeBuildGhostIssue = async (): Promise<IssueCreate | false> => {
```

Comment:
Is this a common pattern in frontend to return IssueCreate | false instead of  IssueCreate | undefined?

Reply:
No. And I think return `IssueCreate | undefined` is better

Reply:
Updated